app/models/user.rb

```diff
@@ -34,6 +34,12 @@ class User < ApplicationRecord
            foreign_key: :reviewer_id,
            dependent: :destroy,
            inverse_of: :reviewer
+  has_many :users_projects, dependent: :destroy
+  has_many :projects, through: :users_projects
+  has_many :pull_requests,
```

Comment:
Can we rename this to `created_pull_requests`? It would better reflect the nature of the relationship

Reply:
Alright!